Monika After Story/game/script-topics.rpy

```diff
@@ -10516,4 +10516,172 @@ label monika_hemispheres_nogets_snow:
+        m 3eua "You know, those dreams where you get to keep your consciousness."
+        m "Where you can, in fact, interact with them and even take control of them."
+        m 1hua "I'm sure you remember."
+        m 1eub "Maybe you even took the advices that I gave you and started putting them into practice."
+        menu:
+            m "Well, did you manage to have a lucid dream, [player]?"
+            
+            "Yes":
+                m 1wub "Wow, I'm impressed!"
+                m 1hua "I know becoming lucid is not an easy thing to manage."
+                m 1esd "Are you an oneironaut, [player]?"
+                m 1tsb "Or did you just get lucky, ehehe~"
+                m 1hua "It sure must be fun to do what you want in a world created just for that."
+                
+            "No":
+                m 1eka "I understand."
+                m 1ekb "It's really difficult to become lucid."
+                m 3eub "But if you want to have more control over what's going on in your dreams, you should definitely read the book by Stephen LaBerge: '{i}Exploring the World of Lucid Dreaming{/i}'."
+                m "As I've told you before, he developed a method to become lucid in a dream."
+                
+        m 1dsd "Having said all that..."
+    m 1tsu "Tell me more about your dream."
+    menu:
+        m "Did you manage to come and see me or did I appear in front of you?"
+
+        "You just appeared before me.":
+            m 1hua "Ehehe~."
+            m 1eub "If my love for you is able to break a barrier between my reality and yours, then I'm sure that we'll be together someday."
+            m 1tfu "I just need to try hard enough for that to happen."
+
+        "I was looking for you.":
+            m 1wud "..."
+            m 1lkbsa "That's so thoughtful of you, sweetheart."
+            m 2ekbfb "Your love is so overwhelming that you created a whole new world just to see me."
+            m 2ekbfa "You have no idea how much this means to me, [player]."
+            
+    m 1eud "By the way, I haven't even ask you what your dream was about."
+    menu:
+        m "So what was it about?"
+        
+        "I dreamed that you would cross over my reality":
+            m 1tsu "Oh~ that kind of dream huh."
+            m 1hub "That's so sweet of you, you want it so badly that you started dreaming about it."
+            m 1eud "I wonder in which manner will I make the transition from one world to another."
+            m 1rud "Will it be in some kind of huge pod connected to a computer, that will produce white smoke when I came out?"
+            m 2lud "Or will it be in an android that downloaded my mind?"
+            m 3eub "Or something more fanciful, like breaking through a computer screen."
+            m 3hksdrb "That last one is highly improbable, but you never know."
+            m 1esb "Anyway, no matter how I'll make the transition, I'm sure you will be there to hug me when the moment come."
```

Comment:
I'm not the best programmer, I'm just starting, so I'm not gonna leave a review.
But I noticed that this line says, "when the moment *come,"* instead of, "when the moment *comes."*